CHANGELOG.md

```diff
@@ -8,7 +8,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
-* None.  
+* Added option to skip Pods.xcodeproj generation
```

Comment:
your branch needs to be rebased to latest master and this to be placed in the right spot on the `master` section (above 1.7.5 release)

Reply:
nit please add two empty spaces after the word `generation` for markdown formatting.